dodo.py

```diff
@@ -5,92 +5,45 @@
 import json
 
 from doit.tools import LongRunning
-from subprocess import check_output, CalledProcessError, PIPE
+from functools import lru_cache
```

Comment:
What is the lru_cache property for?  Isn't there a simpler way to do whatever it is?

Reply:
It caches the result of those properties, so after it is calculated once, it is never calculated again.  There is no cheaper way to provide this functionality.  Also the docs: https://docs.python.org/3/library/functools.html